app/models/coin.rb

```diff
@@ -254,15 +254,15 @@ def hourly_prices_data
   end
 
   def prices_data
-    Rails.cache.fetch("coins/#{id}/prices", expires_in: seconds_to_next_day) do
+    Rails.cache.fetch("coins/#{id}/prices", expires_in: seconds_to_next_day + 1800) do
```

Comment:
What's the logic for the 1800 magic number? Would be helpful to get a comment explaining the situation.

Might also be helpful to create a constant and use it for the `sparkline` call as well if the logic is the same.

Reply:
I thought about it, but that constant by itself wouldn't make so much sense I think? I've added a comment that hopefully helps explain it.
I may adjust the value in the future if it proves to be insufficient.